packages/dropdown/sync/sp-dropdown.ts

```diff
@@ -24,5 +24,10 @@ Dropdown.openOverlay = async (
     content: HTMLElement,
     options: OverlayOptions
 ): Promise<() => void> => {
-    return await Overlay.open(target, interaction, content, options);
+    return await Overlay.open(target, interaction, content, {
+        ...options,
+        openCloseDetail: {
```

Comment:
Now that we're not relying on custom data here, should we close over the API by allowing the internals to always set the value of https://github.com/adobe/spectrum-web-components/pull/1059/files#diff-c6eb6d3cfbad5923aeba984b9ed3a1f7fa7bf8d173860247d4d346fb4726c9ffR370, etc. so that it doesn't accidentally get used before we're able to more formally structure it?

Reply:
Good catch. Change made.

Reply:
I don't think I was clear. With the following being the current payload of the `open` and `close` events:
```
detail: {
    interaction: details.interaction
},
```
Do we need to add `openCloseDetail` to the options bag on the `open()` method at this time? Maybe there's something you're using that for that I'm missing?

Also, does this want the same simplicity as the above usage? https://github.com/adobe/spectrum-web-components/pull/1059/files#diff-c6eb6d3cfbad5923aeba984b9ed3a1f7fa7bf8d173860247d4d346fb4726c9ffR370

Reply:
Sorry for that. I think you were clear, I just didn't go back through as carefully as I should have, which meant I wasted some of your time. Sorry about that! I think this is now carved down to the simple change it should have been :-)